byoqm/main.py

```diff
@@ -22,48 +17,15 @@ def parse_src_root() -> Path:
     return path_to_src
 
 
-def load_quality_model_from_args() -> QualityModel:
-    if not Path(MODELS_PATH).exists():
-        print(f"Make sure the models folder exists")
-        exit(1)
-
-    quality_model_name = sys.argv[2]
-    spec = importlib.util.spec_from_file_location(
-        "model", MODELS_PATH + "/" + quality_model_name + ".py"
-    )
-    module = importlib.util.module_from_spec(spec)
-    sys.modules["model"] = module
-    spec.loader.exec_module(module)
-    # assumes that the python module has a variable model, that is an instantiation of the model class
-    return module.model
-
-
-def save_to_csv(quality_model, out="./output"):
-    file_location = out + "/" + str(date.today()) + ".csv"
-    if not os.path.exists(out):
-        os.mkdir(out)
-    with open(file_location, "w") as file:
-        writer = csv.writer(file)
-        writer.writerow(["metric", "value"])
-        for metric, path in quality_model.getDesc()["metrics"].items():
-            cmd = [f"./{path}", f"{src_root}"]
-            process = subprocess.run(cmd, stdout=subprocess.PIPE)
-            result = process.stdout.decode("utf-8").strip()
-            writer.writerow([metric, result])
-
-
 if __name__ == "__main__":
     if len(sys.argv) < 3:
         print(
             f"Make sure to provide the path to source code and a name of a quality model to use currently have {sys.argv}"
         )
         exit(1)
+    path = Path("byoqm")
+    runner: Runner = Runner("code_climate", path)
```

Comment:
is this the final decision as to how users changes models, or do we later move this to a parameter for a CLI command? Doesn't stop from merging.

Reply:
I Think we should discuss this tomorrow. Some of these parameters are optional, but this one should be required